typing/src/main/java/org/navgurukul/typingguru/utils/Utility.kt

```diff
@@ -2,14 +2,26 @@ package org.navgurukul.typingguru.utils
 
 import android.content.Context
 import android.hardware.usb.UsbManager
-import org.navgurukul.typingguru.utils.Logger.d
+import timber.log.Timber
 import java.util.Random
 import java.util.concurrent.TimeUnit
 
-object Utility {
-    private const val TAG = "Utility"
-    const val TYPE_PRACTICE_TYPING = "practicetyping"
+class Utility {
     private var keyMap: HashMap<String, Int> = HashMap()
+    val TYPE_PRACTICE_TYPING = "practicetyping"
+
+    private val alphabetList: ArrayList<String> by lazy {
+        ArrayList()
+    }
+
+    fun getAlphabets() : ArrayList<String> {
+        var c = 'a'
+        while (c <= 'z') {
+            alphabetList.add(c.toString())
+            ++c
+        }
+        return alphabetList;
+    }
```

Comment:
This would add duplicate values to the list if the function is called multiple times. You can either move this logic inside lazy block and expose the variable or you can remove the lazy block altogether and just use the function to return new list everytime